Source/WebCore/css/typedom/transform/CSSScale.cpp

```diff
@@ -57,6 +61,53 @@ ExceptionOr<Ref<CSSScale>> CSSScale::create(CSSNumberish x, CSSNumberish y, std:
     return adoptRef(*new CSSScale(z ? Is2D::No : Is2D::Yes, WTFMove(rectifiedX), WTFMove(rectifiedY), WTFMove(rectifiedZ)));
 }
 
+ExceptionOr<Ref<CSSScale>> CSSScale::create(CSSFunctionValue& cssFunctionValue)
+{
+    auto makeScale = [&](const Function<ExceptionOr<Ref<CSSScale>>(Vector<RefPtr<CSSNumericValue>>)>& create) -> ExceptionOr<Ref<CSSScale>> {
```

Comment:
Can we make the parameter a Vector&& to prevent hard-to-see unnecessary vector copies?

Reply:
Addressed in 3bf495845111.